content/articles/services.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: Services
+excerpt: 
+categories:
+- Services
+---
+
+# Services
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+## What are services
+
+Services are DNS snippets ready for you to use. Applying a service creates all the DNS configuration needed to serve your webpage or configure your email.
+
+We currently support more than 30 services! Here are the most recognized ones:
+
+* Amazon Web Services
+* Cloudflare
+* GitHub pages
+* Google Apps
```

Comment:
I'd suggest linking to the article on the Google Apps service: http://support.dnsimple.com/articles/google-apps-service/